spec/requests/v1/users/confirmations_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Confirmations', type: :request do
+  let(:user) { create(:user) }
+  let(:user_data) { { id: user.id, email: user.email, created_at: user.created_at } }
+  let(:email_token) { Api::V1::Lib::Service::EmailToken.encode(user_data) }
+  let(:params) { { email_token: email_token } }
+
+  describe 'GET /api/v1/users/confirmation' do
+    before { get "/api/v1/users/confirmation?email_token=#{email_token}", as: :json }
+
+    describe 'Succes' do
+      it 'renders OK' do
+        expect(response).to be_ok
+        expect(JSON.parse(response.body)).to be_empty
+        expect(User.find(user.id)).to be_confirmed
```

Comment:
Request test is about http statuses and rendered context expectations. What are you doing here?

Reply:
I agree, my bad!